src/test/fuzz/util.h

```diff
@@ -259,7 +297,7 @@ auto ConsumeNode(FuzzedDataProvider& fuzzed_data_provider, const std::optional<N
 {
     const NodeId node_id = node_id_in.value_or(fuzzed_data_provider.ConsumeIntegral<NodeId>());
     const ServiceFlags local_services = ConsumeWeakEnum(fuzzed_data_provider, ALL_SERVICE_FLAGS);
-    const SOCKET socket = INVALID_SOCKET;
+    auto sock = std::make_shared<FuzzedSock>(fuzzed_data_provider);
```

Comment:
c702f77a1f5e5988a0c7cee7ef3f7cfc63a9ea1b feel free to ignore: not sure if there is a reason to not keep it const (and use braced initialization)
```suggestion
    const auto sock{std::make_shared<FuzzedSock>(fuzzed_data_provider)};
```

Reply:
Yes, it can be `const`, this is the `std::shared_ptr` itself, not the contained `FuzzedSock`.

Braced initialization is pointless with `auto` since there is nothing to check against. I find braced initialization less readable than `=` so I left it as it is.